<!-- roslaunch db_smf_uwb run_sim.launch -->

<launch>
    <node pkg="db_smf_uwb" type="db_smf_node" name="db_smf_node" output="screen"/>
    <rosparam command="load" file="$(find db_smf_uwb)/config/sim.yaml" />

    <param name="/use_sim_time" value="true"/>
    <arg name="bag_file" default = "$(find db_smf_uwb)/bag/sim_uwb.bag" />
    <node pkg="rosbag" type="play" name="rosbag_play" args="--clock -r 2 $(arg bag_file)"/>

    <node pkg="rviz" type="rviz" name="rviz" respawn="true" args="-d $(find db_smf_uwb)/config/db_smf.rviz" />

</launch>
